--- a/frontend/frontend_base.h	2026-07-17 21:24:33
+++ b/frontend/frontend_base.h	2026-07-17 21:24:33
@@ -342,6 +342,7 @@
             // Add to the flat index of nets
             net->udata = int(net_flatindex.size());
             net_flatindex.push_back(net);
+            net_old_indices.emplace_back();
             // Add to the module-level index of netsd
             midx = net->udata;
             // Create aliases for all possible names
@@ -508,6 +509,7 @@
                                                         impl.get_vector_bit_constval(bits, i));
                     cnet->udata = int(net_flatindex.size());
                     net_flatindex.push_back(cnet);
+                    net_old_indices.emplace_back();
                     net_ref = cnet->udata;
                 } else {
                     // Otherwise, lookup (creating if needed) the net with given in-module index
